### PR TITLE
Notification event urls struct

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Small refactor

## Type of change

- Refactor (refactoring code, removing useless files)

## Testing steps

N/A

## Checklist
* [x] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: producer/servicelog/service_log_producer.go [hunk 1]
```diff
@@ -74,13 +74,13 @@ func (producer *Producer) ProduceMessage(msg types.ProducerMessage) (partitionID
 	req, err := http.NewRequest(http.MethodPost, serviceLogURL, bytes.NewBuffer(msg))
 	req.Header.Add("Authorization", "Bearer "+producer.AccessToken)
 	if err != nil {
-		log.Error().Err(err).Msg("Got error while setting up HTTP request")
+		log.Error().Err(err).Msg("Error setting up HTTP request")
```
Comment (tisnik) [inline producer/servicelog/service_log_producer.go:77]: I'd add the `serviceLogURL`

**Review by tisnik — APPROVED** [review]
LGTM

File: producer/servicelog/service_log_producer.go (hunk 1 above)
Comment (juandspy) [reply producer/servicelog/service_log_producer.go:77]: +1

**Review by juandspy — APPROVED** [review]

File: producer/servicelog/service_log_producer.go (hunk 1 above)
Comment (epapbak) [reply producer/servicelog/service_log_producer.go:77]: on it

Comment (epapbak) [reply producer/servicelog/service_log_producer.go]: Added

**Pull request merged**